huaweicloud/services/cbr/resource_huaweicloud_cbr_policy.go

```diff
@@ -50,6 +58,7 @@ func ResourceCBRPolicyV3() *schema.Resource {
 				ValidateFunc: validation.StringInSlice([]string{
```

Comment:
delete validation?

Reply:
for all regions, there are only backup and replication type of policies they have, it is different with the volume type